toolz/tests/test_dicttoolz.py

```diff
@@ -1,17 +1,41 @@
-from toolz.utils import raises
+import sys
+import collections
+
 from toolz.dicttoolz import (merge, merge_with, valmap, keymap, update_in,
                              assoc, dissoc, keyfilter, valfilter, itemmap,
                              itemfilter)
 
+# Python 2.6 compatibility
+_OrderedDict = getattr(collections, 'OrderedDict',
+                       type('OrderedDict', (dict,), {}))
+PY26 = not all(i <= j for i, j in zip((2, 7), sys.version_info[:2]))
```

Comment:
Would this work?

    PY26 = sys.version_info <= (2, 6)

If so then we might just want to inline this.